pkgs/development/python-modules/poetry/default.nix

```diff
@@ -56,6 +56,11 @@ buildPythonPackage rec {
   postPatch = ''
     substituteInPlace pyproject.toml \
       --replace 'crashtest = "^0.3.0"' 'crashtest = "*"'
+  ''
+  # Track removing the substitution below in https://github.com/python-poetry/poetry/issues/6928
+  + lib.optionalString stdenv.isDarwin ''
+    substituteInPlace pyproject.toml \
```

Comment:
```suggestion
```
Relaxing version constraints in pythonPackages is not worth the comment. Also we want to do this all the time because conditional patching bitrots faster.